docs/index.rst

```diff
@@ -0,0 +1,134 @@
+
+Q-CTRL Open Controls
+====================
+
+Q-CTRL Open Controls is an open-source Python package that makes it easy to
+create and deploy established error-robust quantum control protocols from the
+open literature. The aim of the package is to be the most comprehensive library
+of published and tested quantum control techniques developed by the community,
+with easy to use export functions allowing users to deploy these controls on:
+
+
+* Custom quantum hardware
+* Publicly available cloud quantum computers
+* The `Q-CTRL product suite <https://q-ctrl.com/products/>`_
+
+Anyone interested in quantum control is welcome to contribute to this project.
+
+Table of Contents
+-----------------
+
+
+* `Installation <#installation>`_
+* `Usage <#usage>`_
+* `Contributing <#contributing>`_
```

Comment:
Doesn't look like there are "Contributing" or "Credits" sections below (but there is a "Reference")

Reply:
Gagh sorry. I clearly rushed this one.